### ISSUE TITLE
The wrong of the SiriusAdapter in KNIME

### ISSUE THREAD

**Issue by hello-lei** [OpenMS/OpenMS#6286]
Hello, 
all, there is a wrong about the SiriusAdapter workflow in KNIME. 
Thank you very much!
And I have a question about this. Because I used it to process my data successfully, I would like to know what happen about this?
Thank you.
![image](https://user-images.githubusercontent.com/83147334/183439472-ded2231f-c2d1-4d38-9db6-a2165eb06e5e.png)

Progress of 'loading spectra list':

-- done [took 0.16 s (CPU), 0.16 s (Wall)] -- 
Progress of 'loading chromatogram list':

-- done [took 0.00 s (CPU), 0.00 s (Wall)] -- 
Number of features to be processed: 191
No MS1 spectrum for this precursor. Occurred 0 times.
0 spectra were skipped due to precursor charge below -1 and above +1.
Mono charge assumed and set to charge 1 with respect to current polarity 9 times.
8 features were skipped due to feature charge below -1 and above +1.
Running SIRIUS with the following command line parameters: 
--processors=1 --input C:/Users/leiyuanluo/AppData/Local/Temp/20220808_213347_LAPTOP-95OADKP6_8272_2.ms --project C:/Users/leiyuanluo/AppData/Local/Temp/20220808_213347_LAPTOP-95OADKP6_8272_1/sirius_out sirius --candidates=5 --ppm-max=10.0 --ppm-max-ms2=10.0 --tree-timeout=0 --compound-timeout=100 --profile=qtof fingerid --db=BIO --candidates=10 
Executable is: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat
COMMAND: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat --processors=1 --input C:/Users/leiyuanluo/AppData/Local/Temp/20220808_213347_LAPTOP-95OADKP6_8272_2.ms --project C:/Users/leiyuanluo/AppData/Local/Temp/20220808_213347_LAPTOP-95OADKP6_8272_1/sirius_out sirius --candidates=5 --ppm-max=10.0 --ppm-max-ms2=10.0 --tree-timeout=0 --compound-timeout=100 --profile=qtof fingerid --db=BIO --candidates=10
Executing: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat
Error: Unexpected internal error (Sirius was executed, but an empty output was generated)

**Comment by axelwalter** [comment]
Hi @hello-lei 
Please try the SiriusAdapter with this workflow and your data: https://nodepit.com/workflow/openms.de/Metabolite_DeNovoID
If you data is not centroided yet you can implement one PeakPickerHiRes node right after the InputFile node.

**Comment by hello-lei** [comment]
ok，I will try it.

**Comment by axelwalter** [comment]
Which Sirius version do you use? With Sirius 5 you will need to log in to their online services, otherwise an empty result will be produced.

**Comment by hello-lei** [comment]
No, It still report errors.
Warning: Parameters file version (2.4.0-nightly-2019-01-26) does not match the version of this tool (2.7.0).
Your current parameters are still valid, but there might be new valid values or even new parameters. Upgrading the INI might be useful.
Debug level (after ini file): 2
Value of string option 'no_progress': 0
Value of string option 'executable': sirius.bat
Input file resolved to 'D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat'
Checking input file 'D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat'
Value of string option 'in': F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-HighR_4-257-10701\000\000\HighResPrecursorMassCorrector_0\Volunteer1_blood_10h_POS.mzML
Checking input file 'F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-HighR_4-257-10701\000\000\HighResPrecursorMassCorrector_0\Volunteer1_blood_10h_POS.mzML'
Value of string option 'out_sirius': F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Siriu_4-10-10703\000\000\SiriusAdapter_0\Volunteer1_blood_10h_POS.mzTab
Checking output file 'F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Siriu_4-10-10703\000\000\SiriusAdapter_0\Volunteer1_blood_10h_POS.mzTab'
Value of string option 'out_fingerid': F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Siriu_4-10-10703\000\000\SiriusAdapter_1\Volunteer1_blood_10h_POS.mzTab
Checking output file 'F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Siriu_4-10-10703\000\000\SiriusAdapter_1\Volunteer1_blood_10h_POS.mzTab'
Value of string option 'in_featureinfo': F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Metab_4-11-10700\000\000\MetaboliteAdductDecharger_1\Volunteer1_blood_10h_POS.featureXML
Checking input file 'F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Metab_4-11-10700\000\000\MetaboliteAdductDecharger_1\Volunteer1_blood_10h_POS.featureXML'
Value of string option 'out_ms': F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Siriu_4-10-10703\000\000\SiriusAdapter_2\Volunteer1_blood_10h_POS.ms
Checking output file 'F:\KNIME_temporary\knime_Metabolite_DeNo10694\fs-Siriu_4-10-10703\000\000\SiriusAdapter_2\Volunteer1_blood_10h_POS.ms'
Value of string option 'out_project_space': 
Value of string option 'converter_mode': 0
Progress of 'loading spectra list':

-- done [took 0.16 s (CPU), 0.15 s (Wall)] -- 
Progress of 'loading chromatogram list':

-- done [took 0.00 s (CPU), 0.00 s (Wall)] -- 
Number of features to be processed: 36
No MS1 spectrum for this precursor. Occurred 0 times.
0 spectra were skipped due to precursor charge below -1 and above +1.
Mono charge assumed and set to charge 1 with respect to current polarity 0 times.
0 features were skipped due to feature charge below -1 and above +1.
Running SIRIUS with the following command line parameters: 
--processors=1 --input C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_2.ms --project C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_1/sirius_out sirius --candidates=5 --ppm-max=10.0 --ppm-max-ms2=10.0 --tree-timeout=0 --compound-timeout=100 --profile=qtof fingerid --candidates=10 
Executable is: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat
COMMAND: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat --processors=1 --input C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_2.ms --project C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_1/sirius_out sirius --candidates=5 --ppm-max=10.0 --ppm-max-ms2=10.0 --tree-timeout=0 --compound-timeout=100 --profile=qtof fingerid --candidates=10
Executing: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat
SiriusAdapterAlgorithm.cpp(383): Deleting temporary directory C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_1. Set debug level to 9 or higher to keep it.
SiriusAdapterAlgorithm.cpp(388): Deleting temporary msfile C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_2.ms. Set debug level to 9 or higher to keep it.
Error: Unexpected internal error (Sirius was executed, but an empty output was generated)
Error occurred in line 230 of file C:\jenkins\ws\openms\RC\openms_release_packaging\9447518b\source\src\utils\SiriusAdapter.cpp (in function: enum OpenMS::TOPPBase::ExitCodes __cdecl TOPPSiriusAdapter::main_(int,const char **)) !

**Comment by hello-lei** [comment]
yes, I use 5.

**Comment by axelwalter** [comment]
Ok, so that might be the reason.
You can login with your executable once you created an account.
https://boecker-lab.github.io/docs.sirius.github.io/account-and-license/

**Comment by hello-lei** [comment]
How to I log in the service？Because of the version?

**Comment by hello-lei** [comment]
Thank you! I try it again.

**Comment by hello-lei** [comment]
![image](https://user-images.githubusercontent.com/83147334/184095320-25df970d-002d-4263-b71c-0c0cc51085f0.png)
Do I according this to login in?

**Comment by axelwalter** [comment]
Yes, open a terminal in the directory where your sirius executable is located and login with your email and password.

**Comment by jpfeuffer** [comment]
Interesting, `D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat`

This is our own Sirius, and we do not ship Sirius 5.

**Comment by jpfeuffer** [comment]
Please run the full Sirius command on your PowerShell/batch and report the output. Otherwise it is just random guessing.

**Comment by hello-lei** [comment]
> PowerShell/batch 
Sorry ,I can't understand deeply. What is meaning about this?

**Comment by axelwalter** [comment]
Please set the debug level to 9 in the SiriusAdapter options in KNIME and run the workflow again. Then copy the command from the output (starting with D://) eg:
```
COMMAND: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat --processors=1 --input C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_2.ms --project C:/Users/leiyuanluo/AppData/Local/Temp/20220811_160401_LAPTOP-95OADKP6_412_1/sirius_out sirius --candidates=5 --ppm-max=10.0 --ppm-max-ms2=10.0 --tree-timeout=0 --compound-timeout=100 --profile=qtof fingerid --candidates=10
Executing: D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat
```
and paste that into the PowerShell / command prompt and run it, then post the output here.

**Comment by hello-lei** [comment]
C:\Users\leiyuanluo>D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat
Can't load log handler "java.util.logging.FileHandler"
java.nio.file.NoSuchFileException: C:\Users\leiyuanluo\.sirius-4.8\sirius.log.0.lck
java.nio.file.NoSuchFileException: C:\Users\leiyuanluo\.sirius-4.8\sirius.log.0.lck
        at java.base/sun.nio.fs.WindowsException.translateToIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsFileSystemProvider.newFileChannel(Unknown Source)
        at java.base/java.nio.channels.FileChannel.open(Unknown Source)
        at java.base/java.nio.channels.FileChannel.open(Unknown Source)
        at java.logging/java.util.logging.FileHandler.openFiles(Unknown Source)
        at java.logging/java.util.logging.FileHandler.<init>(Unknown Source)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
        at java.base/java.lang.reflect.ReflectAccess.newInstance(Unknown Source)
        at java.base/jdk.internal.reflect.ReflectionFactory.newInstance(Unknown Source)
        at java.base/java.lang.Class.newInstance(Unknown Source)
        at java.logging/java.util.logging.LogManager.createLoggerHandlers(Unknown Source)
        at java.logging/java.util.logging.LogManager$4.run(Unknown Source)
        at java.logging/java.util.logging.LogManager$4.run(Unknown Source)
        at java.base/java.security.AccessController.doPrivileged(Unknown Source)
        at java.logging/java.util.logging.LogManager.loadLoggerHandlers(Unknown Source)
        at java.logging/java.util.logging.LogManager.initializeGlobalHandlers(Unknown Source)
        at java.logging/java.util.logging.LogManager$RootLogger.accessCheckedHandlers(Unknown Source)
        at java.logging/java.util.logging.Logger.getHandlers(Unknown Source)
        at java.logging/java.util.logging.Logger.log(Unknown Source)
        at org.slf4j.impl.JDK14LoggerAdapter.log(JDK14LoggerAdapter.java:582)
        at org.slf4j.impl.JDK14LoggerAdapter.info(JDK14LoggerAdapter.java:277)
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:245)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.run(SiriusCLIApplication.java:108)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.main(SiriusCLIApplication.java:53)
INFO    19:46:07 - Sirius Workspace Successfull initialized at: C:\Users\leiyuanluo\.sirius-4.9
INFO    19:46:07 - You run SIRIUS 4.9.0
INFO    19:46:07 - Sirius was compiled with the following ILP solvers: GLPK-v1.7.0 (included), Gurobi-v9.1.1, CPLEX-v12.7.1, COIN-OR-v1.17.3
INFO    19:46:07 - Treebuilder priorities loaded from 'sirius.properties' are: [GUROBI, CPLEX, CLP, GLPK]
Application Core STATIC Block Error!
java.lang.IllegalStateException: Unmapped relationship: 7
        at com.sun.jna.platform.win32.WinNT$SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX.fromPointer(WinNT.java:3033)
        at com.sun.jna.platform.win32.Kernel32Util.getLogicalProcessorInformationEx(Kernel32Util.java:715)
        at oshi.driver.windows.LogicalProcessorInformation.getLogicalProcessorInformationEx(LogicalProcessorInformation.java:61)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.initProcessorCounts(WindowsCentralProcessor.java:149)
        at oshi.hardware.common.AbstractCentralProcessor.<init>(AbstractCentralProcessor.java:73)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.<init>(WindowsCentralProcessor.java:68)
        at oshi.hardware.platform.windows.WindowsHardwareAbstractionLayer.createProcessor(WindowsHardwareAbstractionLayer.java:60)
        at oshi.util.Memoizer$1.get(Memoizer.java:87)
        at oshi.hardware.common.AbstractHardwareAbstractionLayer.getProcessor(AbstractHardwareAbstractionLayer.java:66)
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:285)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.run(SiriusCLIApplication.java:108)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.main(SiriusCLIApplication.java:53)
SEVERE  19:46:07 - Unexpected Error!
java.lang.ExceptionInInitializerError
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.run(SiriusCLIApplication.java:108)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.main(SiriusCLIApplication.java:53)
Caused by: java.lang.RuntimeException: java.lang.IllegalStateException: Unmapped relationship: 7
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:305)
        ... 2 more
Caused by: java.lang.IllegalStateException: Unmapped relationship: 7
        at com.sun.jna.platform.win32.WinNT$SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX.fromPointer(WinNT.java:3033)
        at com.sun.jna.platform.win32.Kernel32Util.getLogicalProcessorInformationEx(Kernel32Util.java:715)
        at oshi.driver.windows.LogicalProcessorInformation.getLogicalProcessorInformationEx(LogicalProcessorInformation.java:61)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.initProcessorCounts(WindowsCentralProcessor.java:149)
        at oshi.hardware.common.AbstractCentralProcessor.<init>(AbstractCentralProcessor.java:73)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.<init>(WindowsCentralProcessor.java:68)
        at oshi.hardware.platform.windows.WindowsHardwareAbstractionLayer.createProcessor(WindowsHardwareAbstractionLayer.java:60)
        at oshi.util.Memoizer$1.get(Memoizer.java:87)
        at oshi.hardware.common.AbstractHardwareAbstractionLayer.getProcessor(AbstractHardwareAbstractionLayer.java:66)
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:285)
        ... 2 more

Exception in thread "Thread-0" java.lang.NoClassDefFoundError: Could not initialize class de.unijena.bioinf.ms.frontend.core.ApplicationCore
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.lambda$configureShutDownHook$1(SiriusCLIApplication.java:73)
        at java.base/java.lang.Thread.run(Unknown Source)
C:\Users\leiyuanluo>D:/KNIME/plugins/de.openms.win32.x86_64_2.7.0.202109131426/payload/bin/Sirius/sirius.bat
Can't load log handler "java.util.logging.FileHandler"
java.nio.file.NoSuchFileException: C:\Users\leiyuanluo\.sirius-4.8\sirius.log.0.lck
java.nio.file.NoSuchFileException: C:\Users\leiyuanluo\.sirius-4.8\sirius.log.0.lck
        at java.base/sun.nio.fs.WindowsException.translateToIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsFileSystemProvider.newFileChannel(Unknown Source)
        at java.base/java.nio.channels.FileChannel.open(Unknown Source)
        at java.base/java.nio.channels.FileChannel.open(Unknown Source)
        at java.logging/java.util.logging.FileHandler.openFiles(Unknown Source)
        at java.logging/java.util.logging.FileHandler.<init>(Unknown Source)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
        at java.base/java.lang.reflect.ReflectAccess.newInstance(Unknown Source)
        at java.base/jdk.internal.reflect.ReflectionFactory.newInstance(Unknown Source)
        at java.base/java.lang.Class.newInstance(Unknown Source)
        at java.logging/java.util.logging.LogManager.createLoggerHandlers(Unknown Source)
        at java.logging/java.util.logging.LogManager$4.run(Unknown Source)
        at java.logging/java.util.logging.LogManager$4.run(Unknown Source)
        at java.base/java.security.AccessController.doPrivileged(Unknown Source)
        at java.logging/java.util.logging.LogManager.loadLoggerHandlers(Unknown Source)
        at java.logging/java.util.logging.LogManager.initializeGlobalHandlers(Unknown Source)
        at java.logging/java.util.logging.LogManager$RootLogger.accessCheckedHandlers(Unknown Source)
        at java.logging/java.util.logging.Logger.getHandlers(Unknown Source)
        at java.logging/java.util.logging.Logger.log(Unknown Source)
        at org.slf4j.impl.JDK14LoggerAdapter.log(JDK14LoggerAdapter.java:582)
        at org.slf4j.impl.JDK14LoggerAdapter.info(JDK14LoggerAdapter.java:277)
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:245)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.run(SiriusCLIApplication.java:108)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.main(SiriusCLIApplication.java:53)
INFO    19:46:31 - Sirius Workspace Successfull initialized at: C:\Users\leiyuanluo\.sirius-4.9
INFO    19:46:31 - You run SIRIUS 4.9.0
INFO    19:46:31 - Sirius was compiled with the following ILP solvers: GLPK-v1.7.0 (included), Gurobi-v9.1.1, CPLEX-v12.7.1, COIN-OR-v1.17.3
INFO    19:46:31 - Treebuilder priorities loaded from 'sirius.properties' are: [GUROBI, CPLEX, CLP, GLPK]
Application Core STATIC Block Error!
java.lang.IllegalStateException: Unmapped relationship: 7
        at com.sun.jna.platform.win32.WinNT$SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX.fromPointer(WinNT.java:3033)
        at com.sun.jna.platform.win32.Kernel32Util.getLogicalProcessorInformationEx(Kernel32Util.java:715)
        at oshi.driver.windows.LogicalProcessorInformation.getLogicalProcessorInformationEx(LogicalProcessorInformation.java:61)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.initProcessorCounts(WindowsCentralProcessor.java:149)
        at oshi.hardware.common.AbstractCentralProcessor.<init>(AbstractCentralProcessor.java:73)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.<init>(WindowsCentralProcessor.java:68)
        at oshi.hardware.platform.windows.WindowsHardwareAbstractionLayer.createProcessor(WindowsHardwareAbstractionLayer.java:60)
        at oshi.util.Memoizer$1.get(Memoizer.java:87)
        at oshi.hardware.common.AbstractHardwareAbstractionLayer.getProcessor(AbstractHardwareAbstractionLayer.java:66)
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:285)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.run(SiriusCLIApplication.java:108)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.main(SiriusCLIApplication.java:53)
SEVERE  19:46:31 - Unexpected Error!
java.lang.ExceptionInInitializerError
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.run(SiriusCLIApplication.java:108)
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.main(SiriusCLIApplication.java:53)
Caused by: java.lang.RuntimeException: java.lang.IllegalStateException: Unmapped relationship: 7
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:305)
        ... 2 more
Caused by: java.lang.IllegalStateException: Unmapped relationship: 7
        at com.sun.jna.platform.win32.WinNT$SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX.fromPointer(WinNT.java:3033)
        at com.sun.jna.platform.win32.Kernel32Util.getLogicalProcessorInformationEx(Kernel32Util.java:715)
        at oshi.driver.windows.LogicalProcessorInformation.getLogicalProcessorInformationEx(LogicalProcessorInformation.java:61)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.initProcessorCounts(WindowsCentralProcessor.java:149)
        at oshi.hardware.common.AbstractCentralProcessor.<init>(AbstractCentralProcessor.java:73)
        at oshi.hardware.platform.windows.WindowsCentralProcessor.<init>(WindowsCentralProcessor.java:68)
        at oshi.hardware.platform.windows.WindowsHardwareAbstractionLayer.createProcessor(WindowsHardwareAbstractionLayer.java:60)
        at oshi.util.Memoizer$1.get(Memoizer.java:87)
        at oshi.hardware.common.AbstractHardwareAbstractionLayer.getProcessor(AbstractHardwareAbstractionLayer.java:66)
        at de.unijena.bioinf.ms.frontend.core.ApplicationCore.<clinit>(ApplicationCore.java:285)
        ... 2 more

Exception in thread "Thread-0" java.lang.NoClassDefFoundError: Could not initialize class de.unijena.bioinf.ms.frontend.core.ApplicationCore
        at de.unijena.bioinf.ms.frontend.SiriusCLIApplication.lambda$configureShutDownHook$1(SiriusCLIApplication.java:73)
        at java.base/java.lang.Thread.run(Unknown Source)
C:\Users\leiyuanluo>

**Comment by hello-lei** [comment]
Is this?

**Comment by jpfeuffer** [comment]
You probably deleted your Sirius User folder. Please run:
`mkdir C:\Users\leiyuanluo.sirius-4.8`

**Comment by jpfeuffer** [comment]
Are using an AMD Ryzen PC? Sounds like Sirius 4.9 is not compatible with Ryzen PCs
https://github.com/oshi/oshi/issues/1554

**Comment by jpfeuffer** [comment]
You can try setting the `project:processors` parameter of Sirius to bypass the check. Otherwise, there is nothing we can do. You have to talk to the Sirius developers or wait for our 3.0.0 release where we might support Sirius 5.

**Comment by hello-lei** [comment]
> mkdir C:\Users\leiyuanluo.sirius-4.8

 It occurs access denied.

**Comment by hello-lei** [comment]
> Sounds like Sirius 4.9 is not compatible with Ryzen PCs

Should I download the lower version Sirius at https://boecker-lab.github.io/docs.sirius.github.io/account-and-license/ ?

**Comment by jpfeuffer** [comment]
You can try but I do not have much hope that lower versions do not have this problem.

**Comment by hello-lei** [comment]
Ok, thank you very much!
So now I just wait the 3.0 version SiriusAdapter. Because of the hot potato.

**Comment by eeko-kon** [comment]
Actually I am dealing with the same issue when I use OpenMS 3.0 (build version) with Sirius < 5.0 (specifically 4.9.15). 
`Error: Unexpected internal error (Sirius was executed, but an empty output was generated)
`
I switched to OpenMS 2.7 and it works perfectly fine @jpfeuffer @axelwalter

**Comment by oliveralka** [comment]
@axelwalter: This thread can probably be closed since SIRIUS4 end-of-life was 31.12.2022.
https://bio.informatik.uni-jena.de/2022/11/sirius-4-end-of-life/
I hope everything works with SIRIUS5.
